src/components/WalletAuthModal/WalletAuthModal.tsx

```diff
@@ -26,7 +26,7 @@ enum EConnectorNames {
   WalletConnect = 'walletconnect',
   WalletLink = 'walletlink',
   Fortmatic = 'fortmatic',
-  Portis = 'portis',
+  // Portis = 'portis',
```

Comment:
It is throwing error while on localhost because it tries to check chain with the chains supported by portis. So, I commented all the portis related code.